MoltenVK/MoltenVK/Utility/MVKEnvironment.h

```diff
@@ -39,10 +39,13 @@
 #	define Vk_PLATFORM_SurfaceCreateInfoMVK		VkMacOSSurfaceCreateInfoMVK
 #endif
 
+/** Macro to adjust the specified Vulkan to include the VK_HEADER_VERSION patch value. */
+#define MVK_VULKAN_API_VERSION_HEADER(api_ver)		VK_MAKE_VERSION(VK_VERSION_MAJOR(api_ver),	\
+                                                                    VK_VERSION_MINOR(api_ver),	\
+                                                                    VK_HEADER_VERSION)
```

Comment:
Technically speaking `VK_MAKE_VERSION` is deprecated, and `VK_MAKE_API_VERSION` should be used instead.

Reply:
Fixed. Thanks.